File: lapindromes.java
import java.util.Scanner;

public class lapindromes {
    public static void main(String[] args) {
        Scanner sc= new Scanner(System.in);
        int t= sc.nextInt();
        if (1<=t && t<=100) {
            for (int i = 0; i < t; i++) {
                String str= sc.next();
                String str1="";
                String str2="";
                if (2<=str.length() && str.length()<=1000) {
                    str1=str.substring(0, str.length()/2);
                    if (str.length()%2==0) {
                        str2=str.substring(str.length()/2, str.length());
                    }else{
                        str2=str.substring((str.length()/2)+1, str.length());
                    }
                    char[] c1=str1.toCharArray();
                    char[] c2=str2.toCharArray();
                    java.util.Arrays.sort(c1);
                    java.util.Arrays.sort(c2);

                    if (java.util.Arrays.equals(c1, c2)) {
                        System.out.println("YES");
                    }else{
                        System.out.println("NO");
                    }
                }
            }
        }
    }
}

File: lapindromes.py
print("Enter the number of test cases")
n = int(input())

test=[]
for i in range(1,n+1):
    print("Enter the test case: ")
    a=input()
    test.append(a)

if n >= 1 and n <=100:
    for i in test:
        string= i
        str1 = ""
        str2 = ""
        if len(string) >=2 and len(string) <= 1000:
           if len(string) % 2 ==0:
               str1 = string[0: len(string)//2]
               str2= string[len(string)//2: ]
           else:
                str1 = string[0: len(string)//2]
                str2 = string[len(string)//2 +1: ]

           l1= list(str1)
           l2= list(str2)
           l1.sort()
           l2.sort()

           s1= str(l1)
           s2= str(l2)
           if s1 == s2:
               print("Yes")
           else:
               print("No")
        else:
            print("String Limit Exceeded")

else:
    print("Number of test cases exceeded")


        

         



